tests/config/envvars.bats

```diff
@@ -14,17 +14,16 @@ function teardown() {
 
 @test "sysbox env vars" {
 
-	run __docker run -e "SYSBOX_IGNORE_SYSFS_CHOWN=TRUE" ${CTR_IMG_REPO}/alpine-docker-dbg:latest chown root:root /sys
+	run __docker run --rm -e "SYSBOX_IGNORE_SYSFS_CHOWN=TRUE" ${CTR_IMG_REPO}/alpine-docker-dbg:latest chown root:root /sys
```

Comment:
Why adding `--rm` to all these instruction? In general, i find it easier to debug testcase failures when the container is kept in the system ...

Reply:
It's because some tests (e.g., our shiftfs tests) assume that they are starting with a clean host. If old containers remain, they may cause the tests to fail.